server/build.gradle

```diff
@@ -39,13 +39,12 @@ dependencies {
 
     implementation "org.apache.spark:spark-launcher_2.12:${sparkVersion}"
     implementation 'io.fabric8:kubernetes-client:5.11.2'
+    implementation 'org.apache.hadoop:hadoop-yarn-client:3.3.1'
+    implementation 'org.apache.hadoop:hadoop-common:3.3.1'
```

Comment:
Extract hadoop version to a variable, so that it is not repeated across different dependencies

Reply:
ok